compareECLFiles.cmake

```diff
@@ -298,7 +299,8 @@ add_test_compareECLFiles(CASENAME msw_model_1
                          SIMULATOR flow
                          ABS_TOL ${abs_tol}
                          REL_TOL ${rel_tol}
-                         DIR model1)
+                         DIR model1
+                         TEST_ARGS --tolerance-pressure-ms-wells=10. --tolerance-wells=1.e-6 --solver-max-time-step-in-days=5.0)
```

Comment:
Great to see this one pass. Have you checked that the first two contstraints are still needed, i.e., could it be that max time step is sufficient alone?